backend/src/test/scala/cromwell/backend/wdl/PureStandardLibraryFunctionsSpec.scala

```diff
@@ -27,4 +27,15 @@ class PureStandardLibraryFunctionsSpec extends FlatSpec with Matchers {
     PureStandardLibraryFunctions.transpose(Seq(Success(inArray))) should be(Success(expectedResult))
   }
 
+  behavior of "length"
+
+  it should "get the right answers" in {
```

Comment:
I actually intended to move those across and forgot... If you're doing a big copy/paste and could maybe possibly hoover that up too...... ☺️